projects/media-viewer/src/lib/annotations/comment-set/comment/comment.component.ts

```diff
@@ -41,20 +52,29 @@ export class CommentComponent implements OnChanges {
   @Input() zoom = 1;
   @Input() index: number;
   @Input() page: number;
+
   @ViewChild('form') form: ElementRef;
-  @ViewChild('textArea') textArea: ElementRef;
+  @ViewChild('editableComment') editableComment: ElementRef<HTMLElement>;
+
+  private subscription: Subscription;
 
-  constructor(
-    private readonly commentService: CommentService
-  ) {
+  constructor(private readonly commentService: CommentService,
+              private readonly annotationEvents: AnnotationEventService) {
     this.MAX_COMMENT_LENGTH = 48;
     this.COMMENT_CHAR_LIMIT = 5000;
+
+    this.subscription = annotationEvents.commentSearch
+      .subscribe(searchString => this.searchString = searchString);
```

Comment:
maybe a little debounce and distinct till change here

Reply:
done